### PR TITLE
Add ru locale.

### PR DESCRIPTION
**Pull request by alek13**

### CODE REVIEW

File: locale/areas.ru.tr
Comment (Niklp09) [inline locale/areas.ru.tr]: your translation `Area closed for factions.` add `members` if possible

File: locale/areas.ru.tr
Comment (Niklp09) [inline locale/areas.ru.tr]: your translation  `List your available territories (or all existing territories if you have the areas privilege).` may try to match the `or all areas if you are an admin` part better

File: locale/areas.ru.tr
Comment (Niklp09) [inline locale/areas.ru.tr]: `Protect your own area=Protect an area and set yourself as the owner of it` doesn't match very well but seems ok for me

File: locale/areas.ru.tr
Comment (Niklp09) [inline locale/areas.ru.tr]: your translation `Loads, converts, and saves a legacy file.` (`the areas from` is missing)

File: locale/areas.ru.tr
Comment (Niklp09) [inline locale/areas.ru.tr]: no `<>` needed

**Review by Niklp09 — COMMENTED** [review]
Left some comments, everything else LGTM. I used deepl for translating your translations back to english. Please note, I don't speak russian - a native speaker should check this too.

**Comment by SmallJoker** [comment]
@alek13 The PR currently waits on your feedback on the suggestions above. Feel free to reject such that are not justified so that we can proceed with this PR towards merge.

**Comment by alek13** [comment]
@SmallJoker / @Niklp09 Sorry, I just forget for it. Will take a look at nearest time

**Comment by ssdaniel24** [comment]
Fixed all remarks above.

File: locale/areas.ru.tr
Comment (alek13) [inline locale/areas.ru.tr:17]: "игроков из" -- прикольно, а то "членов" звучит так себе :)))

File: locale/areas.ru.tr
Comment (alek13) [inline locale/areas.ru.tr:35]: here is not truth, as I see in code.
maybe I'm wrong, but

you may not be an administrator, but have the privilege area

Comment (alek13) [reply locale/areas.ru.tr:35]: @ssdaniel24 , can you double check?

**Comment by ssdaniel24** [comment]
*Force push for fixing my git credentials*

File: locale/areas.ru.tr
Comment (ssdaniel24) [reply locale/areas.ru.tr:35]: Maintainers wrote it like this. Who are we to change it? We are translating only.

Comment (alek13) [reply locale/areas.ru.tr:35]: it could be just a mistake. Why not to fix this?

Comment (ssdaniel24) [reply locale/areas.ru.tr:35]: If it is mistake, it is in repo, not in the translation.

**Comment by alek13** [comment]
@SmallJoker / @Niklp09 fixed.

**Comment by Niklp09** [comment]
Thanks, I'll take a look at your changes in the next days.

**Review by Niklp09 — APPROVED** [review]
Checked your new changes w/ deepl, LGTM. Will merge in a few days unless there are objections.
edit: typo

**Comment by alek13** [comment]
:+1:  Danke schön :)

**Pull request merged**
